File: AmidogsManager/AmidogsManager.Database/Models/Meeting.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace AmidogsManager.Database.Models
{
    public class Meeting
    {
        public int Id { get; set; }
        public string? MeetingTitle { get; set; }
        public int MaxParticpants { get; set; }
        public string? Description { get; set; }
        public string? Location { get; set; }
        public DateTime Date { get; set; }
        [JsonIgnore]
        public virtual ICollection<DogMeeting>? DogMeetings { get; set; }

        public Meeting()
        {
            DogMeetings = new List<DogMeeting>();
        }
    }
}

File: AmidogsManagerPresentation/src/app/shared/models/meeting.ts
export interface DogMeeting {
    // Define las propiedades de DogMeeting según lo que tengas en el modelo C#
    meetingId: number;
    // Añade otras propiedades necesarias
  }

export interface Meeting {
    id: number;
    meetingTitle?: string;
    maxParticpants: number;
    description?: string;
    location?: string;
    date: Date;
    dogMeetings?: DogMeeting[];
  }